flocx_market/conf/api.py

```diff
@@ -13,7 +13,7 @@
     cfg.BoolOpt('enable_ssl_api',
                 default=False),
     cfg.BoolOpt('auth_enable',
-                default=True)
+                default=False)
```

Comment:
We definitely don't want to change this default.

Reply:
got it